src/data.js

```diff
@@ -6912,6 +6912,31 @@ module.exports = [
       'WordPress',
     ],
   },
+  {
+    name: 'Phil Stollery',
+    description: 'Technical writer, recovering developer, lover of Whippets, from Yorkshire.',
+    url: 'https://stollerys.co.uk/setup',
+    twitter: '@phils@social.lol',
```

Comment:
Will this work, or does the site redirect to X?

Reply:
Looks like this failed a validation in the PR checks: https://github.com/wesbos/awesome-uses/actions/runs/5976959754/job/16784742144?pr=1654